mapbox/services/uploads.py

```diff
@@ -150,97 +160,90 @@ def create(self, stage_url, tileset, name=None, patch=False):
+    def list(self, account=None, username=None):
         """List of all uploads
 
         Returns a Response object, the json() method of which returns
         a list of uploads
 
         Parameters
         ----------
-        account: str
-            Account name, defaults to the service's username.
+        username : str
+            Account username, defaults to the service's username.
+        account : str, **deprecated**
+            Alias for username. Will be removed in version 1.0.
 
         Returns
         -------
         requests.Response
         """
-
-        if account is None:
-            account = self.username
-        uri = URITemplate(self.baseuri + '/{account}').expand(
-            account=account)
+        username = self._resolve_username(account, username)
+        uri = URITemplate(self.baseuri + '/{username}').expand(
+            username=username)
         resp = self.session.get(uri)
         self.handle_http_error(resp)
-
         return resp
 
-    def delete(self, upload, account=None):
+    def delete(self, upload, account=None, username=None):
         """Delete the specified upload
 
         Parameters
         ----------
         upload: str
             The id of the upload or a dict with key 'id'.
-        account: str
-            Account name, defaults to the service's username.
+        username : str
+            Account username, defaults to the service's username.
+        account : str, **deprecated**
+            Alias for username. Will be removed in version 1.0.
 
         Returns
         -------
         requests.Response
         """
-
-        if account is None:
-            account = self.username
-
+        username = self._resolve_username(account, username)
         if isinstance(upload, dict):
             upload_id = upload['id']
         else:
             upload_id = upload
-
-        uri = URITemplate(self.baseuri + '/{account}/{upload_id}').expand(
-            account=account, upload_id=upload_id)
+        uri = URITemplate(self.baseuri + '/{username}/{upload_id}').expand(
+            username=username, upload_id=upload_id)
         resp = self.session.delete(uri)
         self.handle_http_error(resp)
-
         return resp
 
-    def status(self, upload, account=None):
+    def status(self, upload, account=None, username=None):
         """Check status of upload
 
         Parameters
         ----------
         upload: str
             The id of the upload or a dict with key 'id'.
-        account: str
-            Account name, defaults to the service's username.
+        username : str
+            Account username, defaults to the service's username.
+        account : str, **deprecated**
+            Alias for username. Will be removed in version 1.0.
 
         Returns
         -------
         requests.Response
         """
-
-        if account is None:
-            account = self.username
-
+        username = self._resolve_username(account, username)
```

Comment:
Just thinking aloud, this design pattern for deprecating a kwarg would make a nice decorator.

Reply:
`@deprecated_kwarg` proof of concept: https://gist.github.com/perrygeo/05ea61c699f8d02af49f04e193ab2655